spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigUtilsTests.java

```diff
@@ -16,29 +16,29 @@
 
 package org.springframework.cloud.kubernetes.fabric8.config;
 
-import io.fabric8.kubernetes.client.server.mock.KubernetesServer;
-import org.junit.ClassRule;
-import org.junit.Test;
```

Comment:
unfortunately, it is me to blame for this change. I missed it in one of the previous PRs.